components/ReceiptPreview.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { capitalize } from '../utils';
+
+function Card(props) {
+  const { header, donation } = props.data;
+  const { donor } = donation;
+  const full_name = `${donor.name} ${donor.middlename} ${donor.surname}`;
+
+  return (
+    <div className="card shadow-xl p-4 mt-5">
+      <div className="grid-container grid grid-cols-8">
+        <div className="item1 col-span-3">
+          <div className="-mt-1">
+            <p className="font-semibold text-2xl">{header.title}</p>
+            <p className="font-semibold text-xl mt-2 txt-orange">
+              {header.subtitle}
+            </p>
+            <div className="w-6 border-t-4 border-black my-3" />
+
+            <div className="text-sm">
+              <p className="font-semibold">
+                {'EIN:'} {header.ein}
+              </p>
+              <p>{header.address.street}</p>
+              <p>{header.address.place}</p>
+              <p>{header.address.city}</p>
+              <p className="txt-orange mt-1">{header.phone}</p>
+              <p className="txt-orange">{header.email}</p>
+            </div>
+          </div>
+        </div>
+        <div className="item2 col-span-5">
+          <div className="border-t-4 border-black pt-6 text-sm">
+            <p className="font-semibold my-6">
+              {'501(c)(3)Organization Donation Receipt'}
+            </p>
+            <p className="font-semibold">{'Donation Date'}</p>
+            <p className="mb-4">{donation.date}</p>
+            <p className="font-semibold">{"Donor's Name"}</p>
+            <p className="mb-4">{full_name}</p>
+            <p className="font-semibold">{"Donor's Mailing Address"}</p>
+            <p>{donor.address.street}</p>
+            <p>{donor.address.place}</p>
+            <p className="mb-4">{donor.address.city}</p>
+            <p className="font-semibold">{"Donor's Phone"}</p>
+            <p className="mb-4">{donor.phone}</p>
+            <p className="font-semibold">{'Donation Value'}</p>
+            <p className="mb-4">
+              {'$ '}
+              {donation.value}
```

Comment:
could we just replace it with:
```{`$ ${donation.value}`}```
?

Reply:
It conflicts with `prettier ESLint rules`, we need to keep it this way. Thanks for all insightful your comments though!

Reply:
Unless we change that rule, which I personally find quite annoying.

Reply:
I'm not having any issues with that - what kind of eslint error do you get?

Reply:
Can't you see the `Vercel` failure when deploying?

Reply:
I'm just checking locally - could you give me a link to vercel failure report? This should work same on local and on deploy.

Reply:
https://github.com/margaritahumanitarian/helpafamily/runs/3685018582?check_suite_focus=true

Reply:
My guess is that you're doing something like this:
```javascript
<p className="mb-4">
   {`$·${donation.value}`}
</p>
```
and it is just complaining because it wants:
```javascript
<p className="mb-4">{`$·${donation.value}`}</p>
```
Do you have prettier formatter installed in your IDE?

Reply:
I follow your advice, but still having error while trying to deploy
https://github.com/theflucs/helpafamily/runs/3688775667?check_suite_focus=true

Reply:
I follow your advice, but still having error while trying to deploy
https://github.com/theflucs/helpafamily/runs/3688775667?check_suite_focus=true
Please, have a look

Reply:
I follow your advice, but still having error while trying to deploy
https://github.com/theflucs/helpafamily/runs/3688775667?check_suite_focus=true
Please, have a look

Reply:
most probably you need to have this formatting:
```
<p className="mb-4">{`$·${
  donation.value
}`}</p>
```
which IDE are you using? Do you have prettier installed?

Reply:
I am using Visual Studio Code, I just realised I had Prettier Code Formatter not ESLint. Thanks, I try this way now.